packages/js/test/plugins/auctionHouseModule/cancelBid.test.ts

```diff
@@ -0,0 +1,171 @@
+import { Keypair } from '@solana/web3.js';
+import test, { Test } from 'tape';
+import { sol } from '@/types';
+import {
+  metaplex,
+  killStuckProcess,
+  createNft,
+  assertThrows,
+  createWallet,
+} from '../../helpers';
+import { createAuctionHouse } from './helpers';
+import { Bid } from '@/plugins/auctionHouseModule';
+
+killStuckProcess();
+
+test('[auctionHouseModule] cancel a Private Bid on an Auction House', async (t: Test) => {
+  // Given we have an Auction House and an NFT.
+  const mx = await metaplex();
+
+  const nft = await createNft(mx);
+  const { client } = await createAuctionHouse(mx);
+
+  // And we put a private bid on that NFT for 1 SOL.
+  const { bid } = await client
+    .bid({
+      mintAccount: nft.address,
+      tokenAccount: nft.token.address,
+      price: sol(1),
+    })
+    .run();
+
+  t.false(bid.canceledAt);
+
+  // When we cancel the given bid.
+  const { bid: canceledBid } = await client.cancelBid({ bid }).run();
+
+  // Then bid receipt has canceled at date.
+  t.ok(canceledBid?.canceledAt);
+
+  // And the trade state returns the fee to the fee payer.
+  const updatedTradeStateFeeBalance = await mx
+    .rpc()
+    .getBalance((canceledBid as Bid).tradeStateAddress);
+
+  t.equal(updatedTradeStateFeeBalance.basisPoints.toNumber(), 0);
```

Comment:
On other tests, I've done this slightly differently.

```suggestion
  // And the trade state returns the fee to the fee payer.
  const bidAccount = await mx.rpc().getAccount(bid.tradeStateAddress);
  t.false(bidAccount.exists, 'bid account no longer exists');
```

Could you do the same in these tests, please?